witness/src/Dscp/Witness/Mempool/Logic.hs

```diff
@@ -129,7 +130,7 @@ readFromMempoolLocked pool lock action = do
 writeToMempool
     :: forall ctx m a
     .  (MempoolCtx ctx m, WithinWriteSDLock)
-    => Pool.Mempool Ids Values (AVLP.AVLChgAccum Ids Values) GTxWitnessed
+    => Pool.Mempool Ids Values (AVLP.AVLChgAccum AvlHash Ids Values) GTxWitnessed
```

Comment:
Please introduce decent type aliases for chgaccums (both), statetx and don't forget undo i've mentioned earlier.

Reply:
@astiunov please fix